code/__DEFINES/mobs.dm

```diff
@@ -610,16 +610,16 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 #define GAS_INHALE_REAGENT_TRANSFER_AMOUNT	7
 
 // Squad ID defines moved from game\jobs\job\squad.dm
-#define NO_SQUAD 0
-#define ALPHA_SQUAD 1
```

Comment:
are you sure about this

Reply:
Yes, i need an assoc list.

Reply:
I meant does anything treat it like an integer

Reply:
no